bonus.md

```diff
@@ -3,11 +3,16 @@
 Edit this file to answer the following questions. Give examples. You may need to do a bit of research. :)
 
 1. Why is Hash.fetch useful?
+  It is useful because you can search through the Hash in any order.  If the key isn't found, an error will be returned which can be useful for you to know where the program is failing.
 
 2. What is the disadvantage of single inheritance, and what's a possible solution to that problem?
+Sometimes, you might want a class to take on the behaviors of multiple classes.  One possible solution is to include a module in order for your class to take on all the behaviors you would like.
 
 3. Why are these classes initialized with an options hash?
+The hashes have a key and a value that make possible for us to populate the data in multiple orders, and there are many options for each of these classes.
 
 4. What is the purpose of the private keyword in a class? What does it protect you from and why is that valuable?
+Private keyword in the class mean that methods within the private section of the class are only available to your class.  It can protect your code from being easily accessible to the outside.
```

Comment:
It doesn't actually make it very difficult to access from outside -- but it does signal to the user that they shouldn't need to use it.  Usually used for internal only helper methods that wouldn't make sense to call on the object on their own.